src/main/java/qna/domain/Question.java

```diff
@@ -72,11 +78,30 @@ public User getWriter() {
         return writer;
     }
 
+    public Answers getAnswers() {
+        return answers;
+    }
+
     public boolean isDeleted() {
         return deleted;
     }
 
     public void setDeleted(boolean deleted) {
         this.deleted = deleted;
     }
+
+    public DeleteHistories delete(User loginUser) throws CannotDeleteException {
+        validAndThrows(loginUser);
+        this.deleted = true;
+
+        DeleteHistories deleteHistories = new DeleteHistories(DeleteHistory.ofQuestion(this));
+        deleteHistories.addAll(answers.delete(writer));
+        return deleteHistories;
+    }
+
+    private void validAndThrows(User loginUser) {
```

Comment:
```suggestion
    private void validate(User loginUser) {
```

메서드명은 동사로 시작하는 것이 관례입니다.
또한 validate한다는 이름에 예외처리를 한다는 의미를 포함하고 있다고 생각되는데 어떻게 생각하시나요?

Reply:
넵 메소드가 두가지의 역할을 하네요 
앞으로 메소드는 하나의 일만 하게끔 유의해서 작성해야겠습니다 ㅎㅎ